docker/confd/templates/behat.yml.tmpl

```diff
@@ -73,6 +73,18 @@ default:
                 - DigidepsBehat\FeatureContext: [{ dbName: api, sessionName: 'digideps' }]
             {{ end }}
 
+        infra:
+            description: Minimal test suite to check infrastructure
+            paths: [ %paths.base%/features ]
+            filters:
+                tags: "@infra"
+            contexts:
+            {{ if exists "/api/database/name" }}
+                - DigidepsBehat\FeatureContext: [{ dbName: {{ getv "/api/database/name" }} }]
+            {{ else }}
+                - DigidepsBehat\FeatureContext: [{ dbName: "api" }]
```

Comment:
Small concern that if the env variable is not set then would this run tests against the wrong database?

Reply:
I believe running against `api` would generally be the intended behaviour. It's how all the other suites are set up.